server/db/shop.js

```diff
@@ -0,0 +1,18 @@
+const connection = require('./connection')
+
+module.exports = { getProductsByGardens }
+
+function getProductsByGardens(gardenId, db = connection) {
```

Comment:
Since this is a function name (not db table-related), it might make more sense to name it "getProductsByGarden()".